sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -30,6 +30,20 @@ template <class To, class From> To cast(From value) {
   return (To)(value);
 }
 
+// Older versions of GCC don't like "const" here
+#if defined(__GNUC__) && (__GNUC__ < 7 || (__GNU__C == 7 && __GNUC_MINOR__ < 2))
+// Names of USM functions that are queried from OpenCL
+constexpr char clHostMemAllocName[] = "clHostMemAllocINTEL";
```

Comment:
Maybe just to define temporary some macro for constexpr/const specificator and undefine it after declaration of names?

Reply:
Good suggestion.